lib/rack/cache/meta_store.rb

```diff
@@ -91,12 +91,12 @@ def store(request, response, entity_store)
       vary = response.vary
       entries =
         read(key).reject do |env, res|
-          (vary == (res['Vary'] || res['vary'])) &&
+          (vary == (res['vary'] || res['vary'])) &&
             requests_match?(vary, env, stored_env)
         end
 
       headers = persist_response(response)
-      headers.delete('Age')
+      headers.delete('age')
```

Comment:
Similar comment here. Is it necessary to delete the same header twice?

Reply:
No, if you can submit a PR to fix this, please do! Thanks!